docs/source/en/model_doc/blip.mdx

```diff
@@ -31,6 +31,10 @@ However, most existing pre-trained models only excel in either understanding-bas
 This model was contributed by [ybelkada](https://huggingface.co/ybelkada).
 The original code can be found [here](https://github.com/salesforce/BLIP).
 
+## Resources
+
+- [Google Colab notebook](https://colab.research.google.com/drive/1lbqiSiA0sDF7JDWPeS0tccrM85LloVha?usp=sharing) on how to fine-tune BLIP for image captioning on a custom dataset
```

Comment:
(nit) please consider adding notebooks to a Github repo like huggingface/notebooks, as currently it's probably part of your personal Google Drive

Reply:
Yes! Should be addressed in e6f4b17